### ISSUE TITLE
Retroarch 1.7.2 Xbox OG Video Issues

### ISSUE THREAD

**Issue by jamallego** [libretro/RetroArch#6634]
thanks for these new release, i have these 2 issues:

- i does not work (for me) in 720p, i have component cables, it only works in 480i, when it loads in 720p, the image looks messy
- when i try to mess wih the settings, the 3 times i try it, it freezes the xbox

These are my xbox settings

- xbox  v1.6 
- HDD 500 IDE, 
- unleash is my dashboard
- component cables

just to let you know, the Retroarch 1.7.0 version works ok in 720p

**Comment by Ryunam** [comment]
Can you upload a picture of how it looks, so as to get a better idea of what is happening?

**Comment by jamallego** [comment]
yeah sure, im uploading 3 pictures, 2 of them are from v1.7.2 and the other one is from v1.7.0

thanks for the quick response
![retroarch v 1 7 1](https://user-images.githubusercontent.com/38788087/39370922-c98a6e68-4a05-11e8-832d-776b8195d643.JPG)
![retroarch v 1 7 2](https://user-images.githubusercontent.com/38788087/39370923-c9a5776c-4a05-11e8-8bab-d93964a4ff99.JPG)
![retroarch v 1 7 0](https://user-images.githubusercontent.com/38788087/39370925-c9bdd492-4a05-11e8-9887-9bbbf1da60b2.JPG)

**Comment by Rocky5** [comment]
Need to change the video buffer to 32bit for v1.6 Xbox. Can be fixed by using CDX as a loader that sets the video mode then loads the Retroarch default.xbe. 
( I made a script for XBMC that will patch cdx to use the certificate of Retroarch and install the cdx loader )

Though that’s a workaround and a permanent fix would be better.

**Comment by Mindtwista** [comment]
i have also this issue.
@rocky5, where can i find your script for xbmc?

**Comment by Rocky5** [comment]
@Mindtwista 
It’s part of my XBMC4Gamers project on my github. 
( it’s in the scripts folder )

**Comment by Mindtwista** [comment]
thanks for the fast reply :)
did you mean the "480p Game Loaders" script?
how do i use it?
btw, big thanks for all your work (xbmc4gamers & xbmc-emustation are awesome)

**Comment by jamallego** [comment]
i have download the latest version of retroarch 1.7.3 for the original xbox, i must report i have the same video issue for the original xbox.....

thanks for your hard work

**Comment by Aftershock84** [comment]
Does anyone here have retroarch on the original xbox I can find a zip.
